api/graphql/schema/repository.graphql

```diff
@@ -17,6 +17,10 @@ type Repository {
         query: String
     ): BugConnection!
 
+    allBugsDetail(
```

Comment:
What is that for?

Reply:
I think this might be from trying something out.
At least it looks like it is not used anywhere.
Will remove it.

Reply:
Removed https://github.com/MichaelMure/git-bug/pull/605/commits/580534f6829bd11007b744362c038edba422ae99